### ISSUE TITLE
usePopover: arrowProps should return role="presentation" and aria-hidden="true"

### ISSUE THREAD

**Issue by majornista** [adobe/react-spectrum#4123]
# 🐛 Bug Report 

To ensure that the `svg` that renders the Popover tip is not exposed to assistive technology as an image with no alternative text,  `arrowProps` returned by `usePopover`  should return `role="presentation"` and `aria-hidden="true"`.

## 🤔 Expected Behavior

Popover tip should not be navigable as an  image with no alternative text using VoiceOver.

## 😯 Current Behavior

Popover tip is navigable as an image with no alternative text using VoiceOver.

## 💁 Possible Solution

In `useOverlayPosition`, update 
https://github.com/adobe/react-spectrum/blob/d7cb26e27cac7b43f6fdff68ce4c10c1185d1bb3/packages/%40react-aria/overlays/src/useOverlayPosition.ts#L201-L206

to include `role="presentation"` and `aria-hidden="true"`:
```jsx
    arrowProps: {
      'aria-hidden': 'true',
      role: 'presentation',
      style: {
        left: position.arrowOffsetLeft,
        top: position.arrowOffsetTop
      }
    },
```

## 🔦 Context

Accessibility Audit

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   | [@adobe/react-spectrum@3.25.0](https://github.com/adobe/react-spectrum/releases/tag/%40adobe%2Freact-spectrum%403.25.0)
| Browser          | Chrome Version 110.0.5481.177 (Official Build) (arm64)
| Operating System | macOS Ventura 13.2.1 (22D68)
